src/tests/ResetPassword-test.js

```diff
@@ -0,0 +1,123 @@
+const mock = require("jest-mock");
+import React from "react";
+import { mount } from "enzyme";
+import expect from "expect";
+import { addLocaleData } from "react-intl";
+import * as actions from "../login/redux/actions/postResetPasswordActions";
+import ResetPasswordMain from "../login/components/LoginApp/ResetPassword/ResetPasswordMain";
+import { Provider } from "react-intl-redux";
+import { createMemoryHistory } from "history";
+import { Router } from "react-router-dom";
+const messages = require("../login/translation/messageIndex");
+addLocaleData("react-intl/locale-data/en");
+
+const baseState = {
+  resetPassword: {
+    email_address: ""
+  },
+  config: { 
+    csrf_token: "csrf-token" 
+  },
+  intl: {
+    locale: "en",
+    messages: messages
+  }
+};
+
+const fakeStore = fakeState => ({
+  default: () => {},
+  dispatch: mock.fn(),
+  subscribe: mock.fn(),
+  getState: () => ({ ...fakeState })
+});
+
+function getFakeState(newState) {
+  if (newState === undefined) {
+    newState = {}
+  }
+  return Object.assign(baseState, newState)
+}
+
+describe("ResetPasswordForm Component,", () => {
```

Comment:
It could be worth to do a quick find/replace for all instances of `ResetPasswordForm` if we are moving away from that name. That way, if we ever wanted to use it elsewhere it would not confuse us here.